packages/loader/container-loader/src/container.ts

```diff
@@ -624,6 +628,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         });
 
         this._deltaManager = this.createDeltaManager();
+        this.internalStorageService = new NullBlobStorageService();
+        this._storageService = new BlobCacheStorageService(this.internalStorageService, this.blobsMap);
```

Comment:
The change itself (in isolation) looks fine, but I do not have full picture on its impact.
First of all, I believe that now this._storageService is never undefined, and thus this.storageService getter should be changed accordingly, and signature on context does not need to include undefined?

But the bigger issue is likely expectations at ContainerRuntime layer where I believe we will start getting this new storage and it gets propagated down the stack (to DDSs, I think).  I do not fully understand mechanism how we switch (and impact and propagation) today from undefined to real storage, and in particular - what layers might be caching original storage and may not see newly set storage on attachment. Maybe it's all works just fine, I just do not have good mental model of that process at the moment.